src/pybind/mgr/dashboard/frontend/src/app/app-routing.module.ts

```diff
@@ -370,18 +370,48 @@ const routes: Routes = [
       {
         path: 'cephfs',
         canActivate: [FeatureTogglesGuardService],
-        data: { breadcrumbs: 'File/File Systems' },
         children: [
-          { path: '', component: CephfsListComponent },
           {
-            path: URLVerbs.CREATE,
+            path: 'fs',
```

Comment:
why do we need to hardcode `fs` here? this will make the route `cephfs/fs` isn't it which looks a bit odd.

Reply:
Both filesystems and NFS nav will be  matched, which was not looking good from UX perspective with `/cephfs` and `/cephfs/nfs`. May be we can check for full path, but:
This approach aligns with the rest fo dashboard:
cephfs:
- /fs
-/nfs

rgw:
-/bucket
-/overview
-/nfs